From 638687d581bd06cac7817226e1777f9412396a42 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 15 Mar 2022 16:49:18 +0200
Subject: [PATCH] [SW-78885] gaudi3: export mem init api

This patch exports memory init api implemented in ROM which can
be used by rest of firmware to perform ARC or other PSOC memory
initialization.

Change-Id: I45793517778baf3b8c44d12503a871ba28cb094e
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 8fbe384..2407381 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -6,3 +6,4 @@
 spi_flash_readl
 dma_copy_image_v2
 security_rom_get_hash
+arcboot_rom_mem_init
